packages/builders/scripts/inter-protocol/replace-electorate-core.js

```diff
@@ -38,18 +38,17 @@ export const defaultProposalBuilder = async ({ publishRef, install }, opts) => {
 const configurations = {
   MAINNET: {
     committeeName: 'Economic Committee',
-    // UNTIL https://github.com/Agoric/agoric-sdk/issues/10194
     voterAddresses: {
-      gov1: 'agoric1gx9uu7y6c90rqruhesae2t7c2vlw4uyyxlqxrx',
-      gov2: 'agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5',
-      gov3: 'agoric14543m33dr28x7qhwc558hzlj9szwhzwzpcmw6a',
+      gov1: 'agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5',
+      gov2: 'agoric13p9adwk0na5npfq64g22l6xucvqdmu3xqe70wq',
+      gov3: 'agoric1zayxg4e9vd0es9c9jlpt36qtth255txjp6a8yc',
```

Comment:
The keys should be the names, not gov1 / gov2 / gov3.

https://github.com/Agoric/agoric-sdk/blob/e1baf4e7e8efaab527d7eadf7305281dac8656c9/packages/vm-config/decentral-main-vaults-config.json#L165-L171